src/js/edu-benefits/components/military-history/RotcHistoryReview.jsx

```diff
@@ -10,16 +10,22 @@ export default class RotcHistoryReview extends React.Component {
         <table className="review usa-table-borderless">
           <tbody>
             <tr>
-              <td>Were you commissioned as a result of senior ROTC?</td>
-              <td>{getLabel(yesNoNA, this.props.data.seniorRotcCommissioned.value)}</td>
+              <td>Are you in a senior ROTC scholarship program right now that pays your tuition, fees, books, and supplies? (Covered under Section 2107 of Title 10, U.S. Code)</td>
+              <td>{showYesNo(this.props.data.seniorRotcScholarshipProgram)}</td>
             </tr>
             <tr>
-              <td>Year of commission:</td>
-              <td>{this.props.data.seniorRotc.commissionYear.value}</td>
+              <td>Were you commissioned as a result of senior ROTC?</td>
+              <td>{getLabel(yesNoNA, this.props.data.seniorRotcCommissioned.value)}</td>
             </tr>
+            {this.props.data.seniorRotcCommissioned.value === 'Y'
+              ? <tr>
+                <td>Year of commission:</td>
+                <td>{this.props.data.seniorRotc.commissionYear.value}</td>
+              </tr>
+              : null}
           </tbody>
         </table>
-        {this.props.data.seniorRotc.rotcScholarshipAmounts.map((scholarship, index) => {
+        {this.props.data.seniorRotcCommissioned.value === 'Y' && this.props.data.seniorRotc.rotcScholarshipAmounts.map((scholarship, index) => {
```

Comment:
I noticed we weren't properly hiding the scholarship fields when you answer No to the conditional question.